commands/utils/git.go

```diff
@@ -254,3 +312,16 @@ func toBasicAuth(token, username string) *http.BasicAuth {
 func getFullBranchName(branchName string) plumbing.ReferenceName {
 	return plumbing.NewBranchReferenceName(plumbing.ReferenceName(branchName).Short())
 }
+
+func loadCustomFormats(formatsArray map[string]string) (CustomFormats, error) {
+	format := CustomFormats{
+		commitTitleFormat:      formatsArray["commitTitle"],
+		branchNameFormat:       formatsArray["branchName"],
+		pullRequestTitleFormat: formatsArray["pullRequestTitle"],
```

Comment:
We better use consts to fetch the above mao keys. The same consts should be used all across the code base.